app/assets/stylesheets/db/controllers/works.scss

```diff
@@ -0,0 +1,21 @@
+body.db-works, body.db-draft_works {
+  form {
+    h4 {
+      text-align: center;
+
+      &.work-fields {
+        margin-bottom: 30px;
+      }
+
+      &.edit-request-fields {
+        margin: 30px 0;
+      }
+    }
+  }
+}
+
+body.db-works-index, body.db-works-season, body.db-works-resourceless {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line